nixpkgs/nixpkgs.bzl

```
@@ -200,6 +201,7 @@ _nixpkgs_package = repository_rule(
         "build_file": attr.label(),
         "build_file_content": attr.string(),
         "nixopts": attr.string_list(),
+        "quiet": attr.bool(),
```

```suggestion
        "quiet": attr.bool(default = False),
```
If I understand @guibou correctly, then his point was to make the default we want explicit.